alpakka-brave/src/main/java/no/sysco/middleware/alpakka/brave/javadsl/Brave.java

```diff
@@ -0,0 +1,38 @@
+package no.sysco.middleware.alpakka.brave.javadsl;
+
+import akka.NotUsed;
+import akka.japi.Pair;
+import akka.stream.BidiShape;
+import akka.stream.FlowShape;
+import akka.stream.Graph;
+import akka.stream.javadsl.Flow;
+import akka.stream.javadsl.GraphDSL;
+import brave.Span;
+import brave.Tracing;
+
+public class Brave {
+
+  public static <T> Graph<FlowShape<T, Pair<T, Span>>, NotUsed> startSpan(final Tracing tracing,
+                                                                          final String spanName) {
+    return Flow.fromGraph(new BraveStartSpanFlowStage<>(tracing, spanName));
+  }
+
+  public static <T> Graph<FlowShape<Pair<T, Span>, T>, NotUsed> finishSpan(final Tracing tracing) {
+    return Flow.fromGraph(new BraveFinishSpanFlowStage<>(tracing));
+  }
+
+  public static <A, B, Mat> Graph<FlowShape<Pair<A, Span>, Pair<B, Span>>, Mat> spanFlow(final Tracing tracing,
```

Comment:
Naming seems little inconsistent: startSpan, finishSpan, spanFlow.
Maybe flowSpan would be more consistent with the other two methods. What do you think?